platform-sdk/swirlds-common/src/test/java/com/swirlds/common/io/filesystem/internal/FileSystemManagerImplTest.java

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
The copyright header is duplicated in this file.

Reply:
thanks! I will address this in case a change is needed or a different PR if not!